src/containers/trending-playlists/store/lineups/collections/sagas.ts

```diff
@@ -0,0 +1,38 @@
+import { call, select } from 'redux-saga/effects'
+import { getUserId } from 'store/account/selectors'
+import { LineupSagas } from 'store/lineup/sagas'
+import { PREFIX, trendingPlaylistLineupActions } from './actions'
+import { getLineup } from './selectors'
+import apiClient from 'services/audius-api-client/AudiusAPIClient'
+import { processAndCacheCollections } from 'store/cache/collections/utils'
+import Collection, { UserCollectionMetadata } from 'models/Collection'
+
+function* getPlaylists({ limit, offset }: { limit: number; offset: number }) {
+  const time = 'week' as 'week'
```

Comment:
wow this type is necessary?

Reply:
yeah, in cases like this TS doesn't know to narrow a string type to a string literal